Sources/PrimerSDK/Classes/Core/Primer/PrimerInternal.swift

```diff
@@ -101,25 +101,18 @@ internal class PrimerInternal {
                 messageType: .error,
                 severity: .error)))
 #endif
-        let bundle = Bundle(identifier: "org.cocoapods.PrimerSDK") ?? Bundle(for: Primer.self)
-        let bundleReleaseVersionNumber = bundle.infoDictionary?["CFBundleShortVersionString"] as? String
-        if bundleReleaseVersionNumber != "2.17.0-rc.9" {
-            events.append(Analytics.Event(
-                eventType: .message,
-                properties: MessageEventProperties(
-                    message: "Wrong release version number (\(bundleReleaseVersionNumber ?? "n/a")) detected.",
-                    messageType: .error,
-                    severity: .error))
-            )
-        } else {
-            events.append(Analytics.Event(
+        
+        let bundleReleaseVersionNumber = Primer.bundle.releaseVersionNumber
```

Comment:
If you look inside `BundleExtension.swift` you will see that using this reverts to the cocoapods bundle ID, via `Bundle.primerFramework`. Think we need to update the code there too.

Poorly named instance method behaving as a static method...

Reply:
Oof, good point. I'm not against having it as a static property ultimately (similar to `Bundle.main`) but we should have a single point of entry. 

With that in mind, I'll remove `Primer.bundle` and update `Bundle.primerFramework`, and point the analytics code at that 👍